packages/frontend/src/components/buy/FundingType.jsx

```diff
@@ -45,14 +49,27 @@ const WrapperImg = styled.div`
     display: flex;
 `;
 
-export const FundingType = ({ icon, link, name, track, provideReferrer }) => {
+export const FundingType = ({
+    icon,
+    link,
+    name,
+    track,
+    disabled,
+    provideReferrer,
+}) => {
     return (
-        <LinkWrap href={link} target='_blank' rel={!provideReferrer ? 'noreferrer' : 'noopener'} onClick={() => track && track()}>
+        <LinkWrap
+            href={link}
+            target="_blank"
+            rel={!provideReferrer ? 'noreferrer' : 'noopener'}
+            onClick={() => track && track()}
+            disabled={disabled}
+        >
             <Wrapper>
                 <WrapperImg title={name}>
-                    <img src={icon} alt="" />
+                    <img src={icon} alt={name} />
                 </WrapperImg>
-                <Button />
+                {disabled ? null : <Button />}
```

Comment:
nit: {disabled ? null : <Button />} -> {!disabled && <Button />}

Reply:
I'd like to keep as it is now.
This way it is more verbose but simpler to understand (no need to make invertion in the mind) and easier to read.